src/AWS.Deploy.Recipes/CdkTemplates/AspNetAppElasticBeanstalkLinux/AppStack.cs

```diff
@@ -88,9 +88,48 @@ internal AppStack(Construct scope, RecipeConfiguration<Configuration> recipeConf
                         Namespace = "aws:elasticbeanstalk:environment",
                         OptionName =  "EnvironmentType",
                         Value = settings.EnvironmentType
+                   },
+                   new CfnEnvironment.OptionSettingProperty
+                   {
+                        Namespace = "aws:elasticbeanstalk:managedactions",
+                        OptionName = "ManagedActionsEnabled",
+                        Value = settings.ElasticBeanstalkManagedPlatformUpdates.ManagedActionsEnabled.ToString().ToLower()
                    }
                 };
 
+            if (settings.ElasticBeanstalkManagedPlatformUpdates.ManagedActionsEnabled)
+            {
+                var beanstalkrole = new Role(this, "BeanstalkServiceRole", new RoleProps
+                {
+                    AssumedBy = new ServicePrincipal("elasticbeanstalk.amazonaws.com"),
+                    ManagedPolicies = new[]
+                    {
+                        ManagedPolicy.FromAwsManagedPolicyName("AWSElasticBeanstalkManagedUpdatesCustomerRolePolicy")
+                    }
+                });

```

Comment:
New created role and the old role, do they have same level of permissions.

Reply:
They are 2 different roles with different permissions